src/ngx_http_lua_accessby.c

```diff
@@ -374,4 +373,20 @@ ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
     return NGX_DECLINED;
 }
 
+static ngx_int_t
+ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
```

Comment:
This is too specific. And we should not handle this in specific phase handlers. Along this route, we'll also have to handle the rewrite phase.